alerter.h

```diff
@@ -0,0 +1,38 @@
+int AlertFarenheitToCelcius(float farenheit,int alertFailureCount,int (*network)(float)) 
+{
+         float celcius = (farenheit - 32) * 5 / 9;
+         int returnCode = network(celcius);
+         if (returnCode != 200) {
+                // non-ok response is not an error! Issues happen in life!
+                // let us keep a count of failures to report
+                // However, this code doesn't count failures!
+                // Add a test below to catch this bug. Alter the stub above, if needed.
+                alertFailureCount += 0;
+                }
+         return alertFailureCount;
+}
+
+int networkAlertStub(float celcius) {
+    printf("Test environment : ALERT: Temperature is %.1f celcius.\n", celcius);
```

Comment:
test stub is used to mimic integration points. so it should return values that might reflect the production environment. This also provides an effective way to test the calling function in different ways based on different return values from this stub.